src/components/cart/CartSidebar/CartSidebar.tsx

```diff
@@ -1,58 +1,99 @@
 import React from 'react'
 import { useCart } from 'src/sdk/cart/useCart'
-import { useCartToggleButton } from 'src/sdk/cart/useCartToggleButton'
 import { useCheckoutButton } from 'src/sdk/cart/useCheckoutButton'
 import Button from 'src/components/ui/Button'
-import { ArrowRight as ArrowRightIcon } from 'phosphor-react'
+import {
+  ArrowRight as ArrowRightIcon,
+  X as XIcon,
+  Truck as TruckIcon,
+} from 'phosphor-react'
+import { Badge } from 'src/components/ui/Badge'
+import Alert from 'src/components/ui/Alert'
+import SlideOver from 'src/components/ui/SlideOver'
+import { useUI } from 'src/sdk/ui'
 
 import CartItem from '../CartItem'
 import OrderSummary from '../OrderSummary'
+import EmptyCart from '../EmptyCart'
 
 import './cart-sidebar.scss'
 
 function CartSidebar() {
   const btnProps = useCheckoutButton()
   const cart = useCart()
-  const toggleProps = useCartToggleButton()
-  const { items, gifts, totalItems, isValidating, subTotal, total } = cart
+  const { displayMinicart, closeMinicart } = useUI()
+
+  const { items, totalItems, isValidating, subTotal, total } = cart
+
+  let onDismissTransition: () => unknown
+  const isEmpty = items.length === 0
 
   return (
-    <div className="cart-sidebar" data-testid="cart-sidebar">
-      <Button {...toggleProps}>Close</Button>
-      <div>Cart Item Detais</div>
-
-      {items.map((item) => (
-        <CartItem key={item.id} item={item} />
-      ))}
-
-      <div>Gifts</div>
-      {gifts.map((item) => (
-        <CartItem key={item.id} item={item} />
-      ))}
-
-      {!!totalItems && (
-        <>
-          <div>Cart Summary</div>
-
-          <OrderSummary
-            subTotal={subTotal}
-            total={total}
-            numberOfItems={totalItems}
-            checkoutButton={
-              <Button
-                data-cart-checkout-button
-                variant="primary"
-                icon={!isValidating && <ArrowRightIcon size={18} />}
-                iconPosition="right"
-                {...btnProps}
-              >
-                {isValidating ? 'Loading...' : 'Checkout'}
-              </Button>
-            }
-          />
-        </>
-      )}
-    </div>
+    <SlideOver
+      isOpen={displayMinicart}
+      onDismiss={closeMinicart}
+      onDismissTransition={(callback) => (onDismissTransition = callback)}
+      size="partial"
+      direction="rightSide"
+      className="cart-sidebar__content"
+    >
+      <div className="cart-sidebar" data-testid="cart-sidebar">
+        <div
+          className={`cart-sidebar__body ${
+            isEmpty ? 'cart-sidebar__body--empty' : ''
+          }`}
+        >
+          <header className="cart-sidebar__header">
+            <div className="cart-sidebar__title">
+              <p className="title-section">Your Cart</p>
```

Comment:
Tweaks title size:
```suggestion
              <p className="title-display">Your Cart</p>
```
Could you also change this element's line-height? `1.35` would be enough to center align it.